indy_common/types.py

```diff
@@ -303,6 +307,58 @@ class ClientPoolConfigOperation(MessageValidator):
     )
 
 
+class ConstraintField(MessageValidator):
+    _base_types = None
+
+    def __init__(self, constraint_entity_cls, constraint_list_cls, **kwargs):
```

Comment:
Why do we need to explicitly pass classes here if we have just 1 variant for each argument: `ConstraintEntityField` and `ConstraintListField`?